Dispatcher/Persistence/UserDefaultsManager.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UserDefaultsManager.swift
```

Comment:
Since this is a Manager file, we would want the functions inside it to be as general as possible (like in the AlamofireManager) so we can use it wherever we need throughout the app.
We don't want to have to create a dedicated function for every data that we will need to save to the user defaults (e.g. if the user saw onboarding or not - will we create a "saveSeenOnboarding" function as well?)